tools/makemanifest.py

```diff
@@ -235,6 +235,9 @@ def main():
     # Get paths to tools
     MAKE_FROZEN = VARS["MPY_DIR"] + "/tools/make-frozen.py"
     MPY_CROSS = VARS["MPY_DIR"] + "/mpy-cross/mpy-cross"
+    if sys.platform == "win32":
+        MPY_CROSS += ".exe"
+    MPY_CROSS = os.getenv("MICROPY_MPYCROSS", MPY_CROSS) # override default if env var exists            
```

Comment:
Comment is a bit superfluous imo, but that's maybe just because I know what getenv does.
Anyway: see the Travis build output: build fails becasue the formatting isn't as expected, there should be two spaces before the comment. Basically you should run tools/codeformat.py before comitting now.

Reply:
> Comment is a bit superfluous imo

good point.  many other instances of os.getenv are void of commenting.  removed comment, rebased.